.github/workflows/stale.yml

```diff
@@ -1,28 +1,43 @@
-name: "Stale label"
+---
+# SPDX-license-identifier: Apache-2.0
```

Comment:
Do we need of these generated comments? my suggestion is a clean yml file

Reply:
IMHO, it's a good practice to have headers on files, those can eventually help on SBOM adoption. The Spanish spellchecker contains the same header